ibllib/pipes/ephys_preprocessing.py

```diff
@@ -232,11 +232,10 @@ def _run(self):
 
         # Run the task QC
         qc = TaskQC(self.session_path, one=self.one, log=_logger)
-        qc.extractor = TaskQCExtractor(self.session_path, lazy=True, one=qc.one)
-        # Extract extra datasets required for QC
-        qc.extractor.data = dsets
-        qc.extractor.extract_data(partial=True)
-
+        qc.extractor = TaskQCExtractor(self.session_path, one=qc.one)
```

Comment:
The lazy option would miss adding the `quiescence` in the dictionary and omit the `self.settings` property